assignments/callbacks.js

```diff
@@ -23,8 +23,12 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
+  return cb(arr[1]);
```

Comment:
You're hardcoding an array index, so it'll always refer to that location. Please refer to this documentation to fully grasp `.length` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/length
Solution:
```
function getLength(arr, cb) {
  // getLength passes the length of the array into the callback. Imagine the callback(cb) function as Gru from despicable me.
  console.log(cb(arr));
}

let arrayLength = function(items) { //this is your minion
  return items.length;
}

getLength(items,arrayLength);
```
Please read this article to give you another view of callbacks.
https://medium.freecodecamp.com/javascript-callbacks-explained-using-minions-da272f4d9bcd